keps/sig-cli/3638-kubectl-plugin-subcommands/README.md

```diff
@@ -273,12 +273,12 @@ in back-to-back releases.
 #### Beta
 
 - Gather feedback from developers and surveys
-- Add integration tests
-- Enable shadowing for all non-existent subcommands
```

Comment:
I believe integration tests are required for beta.

Reply:
I'm also confused by the relative lack of changes for promotion to beta. Shouldn't this feature be enabled by default for beta? If so, where are the corresponding changes to this KEP?

Reply:
@logicalhan  Thanks for review. I've added a new item it beta section to better reflect what promotion to beta means for this feature. This will be enabled by default in beta by just removal of environment variable.